packages/BUILD.bazel

```diff
@@ -0,0 +1,6 @@
+package(default_visibility = ["//visibility:public"])
+
+exports_files([
+    "tsconfig-build.json",
```

Comment:
??

Reply:
We need a way for `ts_library` to centrally consume these files, so we export them from here. We can move this to the `tools` directory, but traditionally it's here to aid in discoverability (i.e. the tsconfig lives in the same part of the package as the packages it configures)

Reply:
Since we have a central tsconfig file, I would expect this to be centralized as well and not under packages/

Reply:
The root tsconfig.json is primarily for IDE consumption, but we can tweak it to be reused, sure.

Reply:
So we can't consolidate, because the root tsconfig depends on having `includes`, `outDir`, and `paths`, whereas the build tsconfig does not. Consolidating them breaks the build. Fortunately though we only have one tsconfig for every part of the build library.